CONTRIBUTING.md

```diff
@@ -71,6 +71,12 @@ and you can (cross-)run the entire test suite using:
 MIRI_TEST_TARGET=i686-unknown-linux-gnu ./miri test
 ```
 
+Tests in ``pass-dep`` need to be run using ``./miri run --dep <filename>``.  
+For example:
+```sh
+./miri run --dep tests/pass-dep/shims/libc-fs.rs
```

Comment:
In terms of ordering this now first explains `./miri run`, then `./miri test`, then `./miri run --dep`. That makes little sense. Can you move the new paragraph up so that it is right after `./miri run`?

Reply:
Sure, this is indeed easier to read.